zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/repo/VFSNotebookRepo.java

```diff
@@ -67,8 +67,15 @@ public VFSNotebookRepo(ZeppelinConfiguration conf) throws IOException {
 
     if (filesystemRoot.getScheme() == null) { // it is local path
       try {
-        this.filesystemRoot = new URI(new File(
-            conf.getRelativeDir(filesystemRoot.getPath())).getAbsolutePath());
+        File file = new File(conf.getRelativeDir(filesystemRoot.getPath()));
+        if (!file.exists()) {
+          logger.info("Notebook dir doesn't exist, create.");
+          Boolean success = file.mkdir();
```

Comment:
Thanks for providing bugfix. 

While `VFSNotebookRepo` supposed to use [Apache commons vfs](https://commons.apache.org/proper/commons-vfs/) API instead of using java.io.File directly, I think directory creation can be done something like 

```
FileObject rootDir = fsManager.resolveFile(getPath("/"));
if (!rootDir.exists()) {
  rootDir.createFolder();
}
```
after fsManager is created.